### DISCUSSION TITLE
Meetup.com "Link Shortener" issue

### DISCUSSION THREAD

**Discussion by eselje** [ThatConference/that.us#553] [General]
I attempted to post my that.us event on Meetup's "This is an online event" URL, but it rejected it saying "This link uses a URL shortener that we don't recognize."  I didn't post this to Issues, as this isn't a problem with the code per se, but Meetup needs to be told that "That.Us" is now a perfectly acceptable place to have online events.

**Comment by theClarkSell** [comment]
@eselje interesting. is that something we need to register with them? or is it a valid domain issue '.us'?

**Comment by eselje** [comment]
I'm pursuing this with Meetup.

**Comment by eselje** [comment]
https://help.meetup.com/hc/en-us/articles/360040609112-Hosting-an-online-event-on-Meetup#troubleshooturl

Submitting a ticket.  (914141)

Looks like this will clear it up.

**Reply by theClarkSell** [reply]
Thank you.

**Comment by eselje** [comment]
Long story short - they're looking into it. They support
https://meet.jit.si/, so if there's a way to get to our activities via a
sub-URL of that then it would work now.

It's not a huge deal of course because you can put the URL in a comment in
MeetUp.

**Reply by theClarkSell** [reply]
not sure what you mean by sub-url? 

Are you trying to embed the activity on meetup so you can just see it running?

**Comment by eselje** [comment]
If the activities on that.us were accessible via a Jitisi url like
https://meet.jit.si/that/<ActivityCode> then it would work until they fix
the actual issue.

All I'm really trying to do it make it easy for the people who follow us on
meetup to find the link. They have this checkbox that says it's an online
event, but won't accept the *that.us <http://that.us>* domain. I can add it
in the comments of course but it's just a little more difficult for the
user to find. No biggie.

[image: image.png]
